server/text_generation_server/models/paged_causal_lm.py

```diff
@@ -333,6 +333,9 @@ def __init__(
             total_num_gpu_blocks=total_num_gpu_blocks,
         )
 
+        # log number of free blocks at init
+        print("[PagedKVCacheManager] number of free blocks: %d" % (len(self.kv_cache_manager.free_blocks)))
```

Comment:
do we have a specific logger for this or just print?

Reply:
We're definitely using prints elsewhere in the code (e.g, https://github.com/IBM/text-generation-inference/blob/main/server/text_generation_server/server.py#L197)